albumentations/augmentations/transforms.py

```diff
@@ -2422,10 +2427,34 @@ def __init__(
         self.gauss_sigma = to_tuple(gauss_sigma, gauss_sigma)
         self.intensity = to_tuple(intensity, intensity)
         self.cutout_threshold = to_tuple(cutout_threshold, cutout_threshold)
+        self.color = (
+            color
+            if color is not None
+            else {
+                "rain": [238, 238, 175],
+                "mud": [20, 42, 63],
+            }
+        )
         self.mode = mode if isinstance(mode, (list, tuple)) else [mode]
+
+        if len(set(self.mode)) > 1 and not isinstance(self.color, dict):
```

Comment:
`and len(self.color) != 2` - check that color provided for both modes

Reply:
I think we don't need it since it is covered by lines 2446-2452. What do you think?